spec/services/pmt_service_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe PmtService do
+  def calc(params)
+    described_class.calculate(**params)
+  end
+
+  it "calculates PMT correctly" do
+    [
+	  [1, 1, 1, 2],
+	  [1000, 0.1, 12, 146.76],
+    [1000, 0.2, 12, 225.26],
```

Comment:
The expected response value present in the original requirements is inaccurate. This one is the correct expected value